# Let the python extension use the path from the pkg-config file
# wpoely86@gmail.com
diff -ur apache-arrow-0.12.0.orig/python/cmake_modules/FindArrow.cmake apache-arrow-0.12.0/python/cmake_modules/FindArrow.cmake
--- apache-arrow-0.12.0.orig/python/cmake_modules/FindArrow.cmake	2019-01-16 04:35:59.000000000 +0100
+++ apache-arrow-0.12.0/python/cmake_modules/FindArrow.cmake	2019-04-05 09:38:00.397707915 +0200
@@ -35,9 +35,11 @@
     message(STATUS "Arrow SO and ABI version: ${ARROW_SO_VERSION}")
     pkg_get_variable(ARROW_FULL_SO_VERSION arrow full_so_version)
     message(STATUS "Arrow full SO version: ${ARROW_FULL_SO_VERSION}")
+    pkg_get_variable(ARROW_INCLUDE_DIRS arrow includedir)
     if ("${ARROW_INCLUDE_DIRS}" STREQUAL "")
       set(ARROW_INCLUDE_DIRS "/usr/${CMAKE_INSTALL_INCLUDEDIR}")
     endif()
+    pkg_get_variable(ARROW_LIBRARY_DIRS arrow libdir)
     if ("${ARROW_LIBRARY_DIRS}" STREQUAL "")
       set(ARROW_LIBRARY_DIRS "/usr/${CMAKE_INSTALL_LIBDIR}")
       if (EXISTS "/etc/debian_version" AND CMAKE_LIBRARY_ARCHITECTURE)
